src/test/java/api/support/builders/ChangeDueDateRequestBuilder.java

```diff
@@ -0,0 +1,53 @@
+package api.support.builders;
+
+import static org.folio.circulation.support.JsonPropertyWriter.write;
+import static org.folio.circulation.domain.representations.ChangeDueDateRequest.DUE_DATE;
```

Comment:
I don't think that API test code should depend upon internal implementation details, as it hides the impact of when they change and if they are incorrect